language/resource.language.sk_sk/strings.po

```diff
@@ -1,28 +1,41 @@
-# Kodi Media Center language file
-# Addon Name: Aeon Nox: SiLVO
-# Addon id: skin.aeon.nox.silvo
 # Addon Provider: SiLVO
+# Translators:
+# Patrik Špaňo <pa4k.xyon@gmail.com>, 2018
+# Miro Valec Valko <mvalec@atlas.sk>, 2018-2019
 msgid ""
 msgstr ""
 "Project-Id-Version: KODI Skins\n"
 "Report-Msgid-Bugs-To: http://trac.kodi.tv/\n"
 "POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: Kodi Translation Team\n"
-"Language-Team: Slovak (Slovakia) (http://www.transifex.com/projects/p/kodi-skins/language/sk_SK/)\n"
+"PO-Revision-Date: 2019-08-06 22:10+0200\n"
+"Last-Translator: Miro \"Valec\" Valko <mvalec@atlas.sk>\n"
+"Language-Team: Slovak (Slovakia) (http://www.transifex.com/teamxbmc/kodi-skins/language/sk_SK/)\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Language: sk_SK\n"
 "Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
+"X-Generator: Poedit 2.2.3\n"
+
+msgctxt "Addon Summary"
+msgid "Created by BigNoid, Modded by SiLVO, and Widgets by Mr. V"
+msgstr "Vytvoril BigNoid, Upravil SiLVO, miniaplikácie vytvoril Mr. V"
+
+msgctxt "Addon Description"
+msgid "Completely redesigned to create a modern look, while retaining the classic Aeon feel."
+msgstr "Kompletne prepracovaný dizajn s moderným vzhľadom pri zachovaní klasického štýlu Aeon."
 
 msgctxt "#31000"
 msgid "Directed by"
 msgstr "Režia"
 
 msgctxt "#31001"
 msgid "Written by"
-msgstr "Scenár"
+msgstr "Autor: "
```

Comment:
Please remove colon from msgstr

Reply:
It's my mistake. Original text "Scenár" is right.
```suggestion
msgstr "Scenár"
```